src/Core/MailTemplates/Handlebars/OrganizationSeatsAutoscaled.html.hbs

```diff
@@ -6,39 +6,30 @@
         <td class="content-block"
             style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; margin: 0; -webkit-font-smoothing: antialiased; padding: 0 0 10px; -webkit-text-size-adjust: none;"
             valign="top">
-            To accommodate new user invitations, your seat count has increased from {{InitialSeatCount}} to {{CurrentSeatCount}}. A
-            prorated charge has been immediately applied to your subscription for the new users. This notification will only be sent
+            To accommodate new member invitations, your seat count has increased from {{InitialSeatCount}} to {{CurrentSeatCount}}. A
+            prorated charge has been immediately applied to your subscription for the new members. This notification will only be sent
             once.
         </td>
     </tr>
-    <tr
-        style="margin: 0; box-sizing: border-box;">
-        <td class="content-block"
-            style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; margin: 0; -webkit-font-smoothing: antialiased; padding: 0 0 10px; -webkit-text-size-adjust: none;"
-            valign="top">
-            To manage your subscription:
-            <ol>
-                <li>Log in to your
-                    <a href="https://vault.bitwarden.com/#/organizations/{{{OrganizationId}}}/settings/subscription">
-                        Web Vault
-                    </a> 
-                    and open your Organization.
-                </li>
-                <li>Open the <b>Settings</b> tab and select <b>Subscription</b> from the left-hand menu.</li>
-                <li>Update your subscription.</li>
-                <li>Click <b>Save</b>.</li>
-            </ol>
-        </td>
-    </tr>
     <tr
         style="margin: 0; box-sizing: border-box;">
         <td class="content-block last"
             style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; margin: 0; -webkit-font-smoothing: antialiased; padding: 0; -webkit-text-size-adjust: none;"
             valign="top">
-            For more information, please refer to
-            <a href="https://bitwarden.com/help/article/managing-users/#subscription">
-                our help article.
+            For more information, please refer to the following help article:
+            <a href="https://bitwarden.com/help/managing-users" class="inline-link">
+                Member management
+            </a>
+            <br style="margin: 0; box-sizing: border-box; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;" />
```

Comment:
Can we include this as a reusable style? It's repeated quite a bit on all templates.

Reply:
Done, but per the comment I added, I've limited the styling to a particular class (even if it looks a little redundant) so that we don't have to mess with & regression test every single email template.